cime_config/config_files.xml

```diff
@@ -127,7 +127,7 @@
       <value component="satm"                        >$SRCROOT/components/stub_comps/satm</value>
       <value component="xatm"                        >$SRCROOT/components/xcpl_comps/xatm</value>
       <value component="eam"                         >$SRCROOT/components/eam/</value>
-      <value component="scream"                      >$SRCROOT/components/scream/</value>
+      <value component="scream"                      >$SRCROOT/components/eamxx/</value>
```

Comment:
Never mind about the above comments if we want to retain the component name 'scream'.

Reply:
Yes these are good points. I think we will want to change these things, but I think that is work for a subsequent PR.